lib/trento/application/process_managers/deregistration_process_manager.ex

```diff
@@ -44,22 +49,47 @@ defmodule Trento.DeregistrationProcessManager do
   """
   def interested?(%HostRegistered{host_id: host_id}), do: {:start, host_id}
   def interested?(%HostRolledUp{host_id: host_id}), do: {:start, host_id}
+  def interested?(%HostAddedToCluster{host_id: host_id}), do: {:start, host_id}
+  def interested?(%ClusterRolledUp{snapshot: %{hosts: hosts}}), do: {:start, hosts}
+
```

Comment:
Whoops, I asked for this to increase the readability of start/stop/continue since it was becoming a big chunk. Maybe we can use comments in the empty lines, not a strong preference tho

Reply:
Allright! as you like folks. Fine for me, let's not extend this hehe